app/src/main/java/org/wikipedia/views/ViewUtil.kt

```diff
@@ -23,11 +27,14 @@ import org.wikipedia.util.DimenUtil.roundedDpToPx
 import org.wikipedia.util.ResourceUtil.getThemedColor
 import org.wikipedia.util.WhiteBackgroundTransformation
 import java.util.*
+import kotlin.math.abs
 
 object ViewUtil {
     private val CENTER_CROP_ROUNDED_CORNERS = MultiTransformation(CenterCrop(),
             WhiteBackgroundTransformation(), RoundedCorners(roundedDpToPx(2f)))
     val ROUNDED_CORNERS = RoundedCorners(roundedDpToPx(15f))
+    const val CLICK_ACTION_THRESHOLD = 200
+    const val CLICK_TIME_THRESHOLD = 400
```

Comment:
Looks like these two variables can be `private`. And would it be possible to move these two variables to the top of the first `private val` variable?